Raport.md

```diff
@@ -1,17 +1,125 @@
-Lego manufacturing facility or storage location. Lego does not usually sell in piece by piece but 
-in different building kits, intended to be assembled by the end user. This means that we, as the 
-provider need to assemble all the different pieces of the many different kits before shipping them 
-to the customer. That means that we need to have an exact count of every piece and know 
-exactly what the different kits require. 
+Lego manufacturing facility or storage location. Lego does not usually sell in piece by piece but in different building kits, intended to be assembled by the end user. This means that we, as the provider need to assemble all the different pieces of the many different kits before shipping them to the customer. That means that we need to have an exact count of every piece and know exactly what the different kits require. 
 
 ## ER Models
 See figure 1
 ![Figure 1](./figures/Lego.png)
 
 ## SQL Queries
 
+### Admin page
+
+On the admin page we display a lot of differet data. We are for example displaying all the orders that has been placed on the Lego Web Store. To get all the orders and it's related relevant data, we use this select statement: 
+```
+SELECT `Order`.id, `Order`.TotalSum, `Order`.Customer, ShippingAdress.StreetAdress, ShippingAdress.PostCode, ShippingAdress.City, `Order`.OrderDate, `Order`.ShippingDate, `Order`.ArrivalDate
+    FROM `Order`
+    CROSS JOIN ShippingAdress ON `Order`.ShippingAdress = ShippingAdress.id
+```
+With this statement we get the Order ID, Total Sum, Customer ID, Shipping Adress, Post Code, City, Order Date, Shipping Date and Arrival Date for every order. Of course it would have been easier to solely display all the data that's available in the `Order` table and nothing else. But then we would be missing some valuble and relavant information for the orders. That's why we also want to get some data from the related `ShippingAdress` table. We want the `StreetAdress`, `PostCode` and the `City` from the `ShippingAdress` table. To correctly align the two tables; `Order` and `ShippingAdress`, we use `CROSS JOIN` on the shipping adress id, which is a value that both tables contains (`Order.ShippingAdress` for the `Order` table, and `ShippingAdress.id` for the `ShippingAdress` table). 
+
+Aside from that select statement above, we also have a query that looks like this, on both the front page and admin page:  
+```
+SELECT * from LegoSet WHERE CheckAvailability(LegoSet.id) = True;
+```
+This statement is used to displays all the available Lego sets at that moment. We use almost the same query for displaying all the unavailable Lego sets; but instead of `CheckAvailability(LegoSet.id) = True`, the unavailable Lego sets is displayed using `CheckAvailability(LegoSet.id) = False` insted. `CheckAvailability` is a function we created ourselves, because we needed a way to check whether our LegoSets were available for purchase or not.
```

Comment:
Replace `CheckAvilability` with [CheckAvailability](#CheckAvailablity), aka link to the heading instead of just marking the word